src/homepageExperience/containers/CliWizard.tsx

```diff
@@ -74,7 +75,9 @@ export class CliWizard extends PureComponent<{}, State> {
           'firstMile.cliWizard.next.clicked',
           {},
           {
-            clickedButtonAtStep: this.state.currentStep - 1,
+            clickedButtonAtStep: normalizeEventName(
+              HOMEPAGE_NAVIGATION_STEPS_CLI[this.state.currentStep - 1].name
+            ),
```

Comment:
previously, this was sending the _index_ of a step in `HOMEPAGE_NAVIGATION_STEPS_CL`. Which is a useless info for eventing. We need to be tracking the step name, not its index. 
Also, I'm not entirely sure why we need to report current step (line 81). I was going to remove it altogether, but thought I should get a second opinion first.